modules/javafx.graphics/src/main/java/com/sun/prism/impl/ps/BaseShaderGraphics.java

```diff
@@ -87,7 +87,7 @@ protected BaseShaderGraphics(BaseShaderContext context,
         this.context = context;
     }
 
-    BaseShaderContext getContext() {
+    BaseShaderContext getBaseShaderContext() {
         return context;
     }
```

Comment:
Mentioning this was brought up in relation to a different PR: https://github.com/openjdk/jfx/pull/628#issuecomment-924898354

Reply:
I think it's a safe change (and this method should probably be made private, as it is only called from the same class).

Reply:
> I think it's a safe change

Agreed.

> (and this method should probably be made private, as it is only called from the same class).

No, it's called from `CachingShapeRep`, which you (necessarily) modified as part of this PR.

Reply:
right, sorry - that's why I did not make it private.

(note to self: double check before responding)